web3.js/babel.config.js

```diff
@@ -0,0 +1,62 @@
+const isCalledByBundler = caller => caller.name.startsWith('@rollup');
+
+module.exports = api => {
+  const calledByBundler = api.caller(isCalledByBundler);
+  const transformEnvVariables =
+    !calledByBundler &&
+    process.env.NODE_ENV &&
+    process.env.BROWSER !== undefined;
+
+  return {
+    presets: [
+      [
+        '@babel/preset-env',
+        {
+          modules: false,
+          bugfixes: true,
+        },
+      ],
+      ['@babel/preset-typescript'],
+    ],
+    plugins: [
+      [
+        '@babel/plugin-proposal-class-properties',
+        {
+          loose: true,
+        },
+      ],
+      [
+        '@babel/plugin-proposal-private-methods',
+        {
+          loose: true,
+        },
+      ],
+      [
+        '@babel/plugin-proposal-private-property-in-object',
+        {
+          loose: true,
+        },
+      ],
+      transformEnvVariables && [
+        'transform-inline-environment-variables',
```

Comment:
This on the other hand, I think is good to keep in any case, since it enables not forcing consumers to inject themselves inlined `process.env.BROWSER` & `process.env.NODE_ENV` values in their builds (even though, `process.env.NODE_ENV` is injected by most bundlers automatically today)